src/nxt_js.c

```diff
@@ -243,13 +243,19 @@ nxt_js_add_tpl(nxt_js_conf_t *jcf, nxt_str_t *str, nxt_bool_t strz)
                                            "    return ");
 
     /*
+     * Appending a newline character if newline is true.
```

Comment:
I would rather s/Appending/Append/, because... English... but then this matches the other one...

Reply:
Good to know.